modules/tools/sonarqube/main.tf

```diff
@@ -3,22 +3,27 @@ resource "random_string" "sonarqube_db_password" {
   special = false
 }
 
-resource "random_string" "sonar_jenkins_password" {
-  length  = 10
-  special = false
-}
+#resource "random_string" "sonar_jenkins_password" {
+#  length  = 10
+#  special = false
+#}
```

Comment:
let's remove this if it's no longer needed

Reply:
sounds good - resource removed